app/views/case_workers/claims/_miscellaneous_fees.html.haml

```diff
@@ -0,0 +1,4 @@
+= govuk_detail "Types of miscellaneous fees you can claim", id: "#misc-fees-list" do
```

Comment:
Could this text be moved into a translation file (probably `config/locales/en.yml`) and then referenced here?

Reply:
Also I'm not sure that you need `id: "#misc-fees-list"` any more, as we're not using to identify the component in the feature tests.